src/atc/orchestrators/ehjson2delta/EhJsonToDeltaOrchestrator.py

```diff
@@ -8,6 +10,7 @@
 )
 
 
+@deprecation.deprecated(details="Currently in preview.")
```

Comment:
is there any other which are non-tested? those should might also have preview in the deprecation details.